src/Microsoft.AspNetCore.Mvc.Abstractions/ApiParameterDescription.cs

```diff
@@ -40,6 +40,6 @@ public class ApiParameterDescription
         /// <summary>
         /// Gets or sets the parameter description.
```

Comment:
Also this is the parameter descrip**tor** it's a reference to the info we cache about the actual method parameters